README.md

```diff
@@ -1,65 +1,73 @@
-README
-======
+<p align="center"><a href="https://symfony.com" target="_blank">
+    <img src="http://symfony.com/logos/symfony_black_02.svg">
```

Comment:
regarding HTTPS on semver.org, this is because they are hosted on github pages, and github pages don't support HTTPS for custom domains (the certificate is valid only for the github.io domains)